lib/cartopy/crs.py

```diff
@@ -959,12 +960,33 @@ def __init__(self, central_longitude=0.0,
             If omitted, a default globe is created.
         latitude_true_scale: optional
             The latitude where the scale is 1. Defaults to 0 degrees.
+        false_easting: optional
+            X offset from the planar origin in metres. Defaults to 0.
+        false_northing: optional
+            Y offset from the planar origin in metres. Defaults to 0.
+        scale_factor: optional
+            Scale factor at natural origin. Defaults to unused.
 
+        Only one of ``latitude_true_scale`` and ``scale_factor`` should
+        be included.
         """
         proj4_params = [('proj', 'merc'),
                         ('lon_0', central_longitude),
-                        ('lat_ts', latitude_true_scale),
+                        ('x_0', false_easting),
+                        ('y_0', false_northing),
                         ('units', 'm')]
+
+        # If it's None, we use the default for Proj4, which is 0.0
```

Comment:
The comment reads as if *we* are going to pass 0.0, but I guess you mean we implicitly use 0.0 because that is what proj4 assumes when we don't use it?

Reply:
That's correct. I've tried to clarify that comment.